src/main/java/com/uwetrottmann/tmdb/services/CollectionService.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2015 Manuel Laggner
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.uwetrottmann.tmdb.services;
+
+import com.uwetrottmann.tmdb.entities.AppendToResponse;
+import com.uwetrottmann.tmdb.entities.Collection;
+import com.uwetrottmann.tmdb.entities.Images;
+import retrofit.http.GET;
+import retrofit.http.Path;
+import retrofit.http.Query;
+
+public interface CollectionService {
+    @GET("/collection/{id}")
+    Collection summary(@Path("id") int var1, @Query("language") String var2, @Query("append_to_response") AppendToResponse var3);
```

Comment:
Please at least copy the method description from the API docs and add it as javadoc. Also, parameters could use some descriptive names. See the other services for examples.